### PR TITLE
Add support for --no-default-features

### PR DESCRIPTION
**Pull request by ashpil**
This implements #114.

### CODE REVIEW

**Comment by ashpil** [comment]
@djc

File: src/bin/cargo-flamegraph.rs [hunk 1]
```diff
@@ -67,6 +67,10 @@ struct Opt {
     #[structopt(short = "f", long = "features")]
     features: Option<String>,
 
+    /// Disable default features
+    #[structopt(short = "n", long = "no-default-features")]
+    no_def_features: bool,
```
Comment (djc) [inline src/bin/cargo-flamegraph.rs:72]: Let's not have a short option for this (Cargo doesn't either), and let's expand the field to the full name.

Comment (ashpil) [reply src/bin/cargo-flamegraph.rs]: Done

**Comment by djc** [comment]
Thanks!

**Pull request merged**